### ISSUE TITLE
We need to know if this has been abandoned, for our safety

### ISSUE THREAD

**Issue by wreaver** [bromite/bromite#2610]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.
- [X] This is a bug report about the Bromite browser (not about the website, building Bromite, F-Droid or anything else).

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

108.0.5359.156

### Device architecture

arm64

### Android version

13

### Device model

SM-A528B

### Changed flags

no changed flags

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

No

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

No

### Describe the bug

There haven't been any updates for months. I don't want to sound rude, but I believe, for our safety, we deserve to know if the project has been abandoned. I seriously appreciate the work that has been put into this project, it's an amazing browser, however, using an outdated browser can pose security risks, so if this project will no longer be updated, please make a formal announcement and let us know.

### Steps to reproduce the bug

Not a bug

### Expected behavior

Not a bug

**Comment by d1stru3t0r** [comment]
Agree, at least we should know to switch to google's updated system webview.

**Comment by HarriBuh** [comment]
It's funny how people still like and download this app even though it's absolutely UNSAFE doing so (due to missing security updates).
It's clearly abandoned, maybe after 6 months (= +1 more month) of silence it will finally be understood by everyone.

**Comment by gigachaand** [comment]
https://github.com/uazo/bromite-buildtools

**Comment by d1stru3t0r** [comment]
> https://github.com/uazo/bromite-buildtools

In using only the system webview which he doesn't have.

**Comment by belieb** [comment]
> It's funny how people still like and download this app even though it's absolutely UNSAFE doing so (due to missing security updates). It's clearly abandoned, maybe after 6 months (= +1 more month) of silence it will finally be understood by everyone.

It's pretty clear to me too. From multiple updates a month to no updates at all starting this year. However, it is confusing that in #2575 from February, the developer said that nobody quit this project and gave the impression in that comment that they were still going to continue it at some point (they didn't explicitly state that though). So that's confusing.

I wouldn't consider it extremely unsafe after just four months, but it soon will be. And it's already unstable, see issue #2600 . It's unusable on all recent Pixel (6 and 7) devices after a recent Android update, it crashes immediately or when using certain menus, causing you to close all tabs and restart the browser. So those users are literally forced to use another browser and they most likely will keep using that. A few months ago, it would've been fixed with an update.

And if you're not forced, at this point, I would recommend anyway to look for something else and to close the chapter of Bromite. Sorry.

**Comment by wreaver** [comment]
> > It's funny how people still like and download this app even though it's absolutely UNSAFE doing so (due to missing security updates). It's clearly abandoned, maybe after 6 months (= +1 more month) of silence it will finally be understood by everyone.
> 
> It's pretty clear to me too. From multiple updates a month to no updates at all starting this year. However, it is confusing that in #2575 from February, the developer said that nobody quit this project and gave the impression in that comment that they were still going to continue it at some point (they didn't explicitly state that though). So that's confusing.
> 
> I wouldn't consider it extremely unsafe after just four months, but it soon will be. And it's already unstable, see issue #2600 . It's unusable on all recent Pixel (6 and 7) devices after a recent Android update, it crashes immediately or when using certain menus, causing you to close all tabs and restart the browser. So those users are literally forced to use another browser and they most likely will keep using that. A few months ago, it would've been fixed with an update.
> 
> And if you're not forced, at this point, I would recommend anyway to look for something else and to close the chapter of Bromite. Sorry.

That reply is something that confuses me as well. They stated back in February that nobody had abandoned anything, yet, there hasn't been a single word since then. Is it abandoned or not then? If I had to speculate, there was probably some problems going on in the background and the project has been quietly abandoned since that reply. Of course, this is just speculation. Hopefully everything's going well with the team, since they made an amazing browser than I'm genuinely saddened is no longer mantained. No other browser will come to be as perfect for me as Bromite.

**Comment by HarriBuh** [comment]
@wreaver  It's speculative to say, but it's likely so that every fork of Chromium>Bromite will be at least as safe and good and Bromite was. And since more and more capable Devs and former fans of Bromite begin to act, it's likely there will be some nice and up-to-date forks soon. Like uazo's own fork.

**Comment by ViktorKahu** [comment]
It has been abandoned.

But you will not hear it "from the horses mouth".

**Comment by heliophane** [comment]
How difficult would it be to take over work of building the browser? I've heard chromium can be a massive pain to compile.

**Comment by sankhababu** [comment]
> How difficult would it be to take over work of building the browser? I've heard chromium can be a massive pain to compile.

That's right, but Uazo going on still now. 
So it's not likely that Carl can't. 
Following the discussion on here, Uazo's built site till the date, I sense there may be some conflict going on between devs.
These peoples are genius and good-hearted. The problem is with such offbeat persons is, they are very sensitive and emotional in nature. I've seen this with other such genius, like dev of NetGuard, Mulch, Mull. 
May time will resolve issues and save us 🙏

**Comment by heliophane** [comment]
> > How difficult would it be to take over work of building the browser? I've heard chromium can be a massive pain to compile.
> 
> That's right, but Uazo going on still now. So it's not likely that Carl can't. Following the discussion on here, Uazo's built site till the date, I sense there may be some conflict going on between devs. These peoples are genius and good-hearted. The problem is with such offbeat persons is, they are very sensitive and emotional in nature. I've seen this with other such genius, like dev of NetGuard, Mulch, Mull. May time will resolve issues and save us pray

Yeah, I discovered uazo's repo of builds last night. I'd be more than happy to throw them a few dollars a month to continue the legacy, maybe under a different name? When was the last time Carl was active?

**Comment by SkewedZeppelin** [comment]
> I wouldn't consider it extremely unsafe after just four months

Already 112 known security issues per my table https://divestos.org/misc/ch-dates.txt

**Comment by xarantolus** [comment]
@heliophane 
> How difficult would it be to take over work of building the browser?

Building the browser itself takes some time to get working, but is definitely possible if you have some experience with Linux and shell scripts.

> I've heard chromium can be a massive pain to compile.

In my opinion it has been one of the better open source projects about this - there's documentation on how to achieve it and that has almost always worked for me.
I have also automated [building Bromite in Docker](https://github.com/xarantolus/bromite-build), so if the bromite patches and my scripts still work (didn't test this for a dozen weeks) it should be as easy as `make bromite` (however you won't get the typical bromite branding, but same functionality)

Edit: even if they worked, it would use the newest (possibly outdated) bromite version as base btw

**Comment by heliophane** [comment]
@xarantolus Consider assisting @uazo on their repo here:
https://github.com/uazo/bromite-buildtools

They're a bromite Dev who has been continuing the project on their own

**Comment by sankhababu** [comment]
> @xarantolus Consider assisting @uazo on their repo here: https://github.com/uazo/bromite-buildtools
> 
> They're a bromite Dev who has been continuing the project on their own

Yes, I'm using this. It's good and with added more features like more content blocking options, clear all data automatically on start etc. 
But Uazo still confused about the future planning of his repo. No automatic update notification.

**Comment by Universalizer** [comment]
> @xarantolus Consider assisting @uazo on their repo here: https://github.com/uazo/bromite-buildtools
> 
> They're a bromite Dev who has been continuing the project on their own

Yes, i think, uazo is alone,
https://github.com/bromite/bromite/issues/2591#issuecomment-1451261573

**Comment by d1stru3t0r** [comment]
Is there any alternative for webview? The other guy maintains only the browser.

**Comment by syphyr** [comment]
> Is there any alternative for webview? The other guy maintains only the browser.

Mulch webview updated by fdroid

**Comment by rookie-ger** [comment]
Although it feels strange that the project has been left behind like this, I send the developers all the best wishes here. Maybe something unexpectedly has gone badly wrong in their lives and I hope they find the power to get back on track...with or without Bromite.

**Comment by vjrmy** [comment]
Why dont you guys using this ungoogled build https://chromium.woolyss.com/#android
Its applied all patches bromite and support ad blocking that can add custom filter

**Comment by vjrmy** [comment]
Oh i see, btw i need this build for armv7. Any fork of Uazo's built for armv7?

**Comment by ghost** [comment]
I'd say yea it is at this point. It's okay to stop working on a project, but the maintainers should at least have made it clear that the project was going to be left behind. 
It's not a good idea to use bromite at this point. 
Given #2141, I wouldn't be surprised if Micay threatened some kind of legal action to stop them from developing, he's done legal threats before. There is no real evidence behind it, just speculation based on my personal experience so yea take it with 16 grains of salt.

**Comment by uazo** [comment]
I ask everyone to please stop writing in this issue, respect people's will. thank you.

**Comment by rookie-ger** [comment]
Come back to topic please and leave people alone that do not have to do with it. Such speculations are extremely harmful.

**Comment by ghost** [comment]
> @sylentpunk That's just speculation, csagan made it clear many times time that there is no legal threat. Daniel was probably just going through his daily bouts of schizophrenia, I believe he's nice and as benign as a Buddha

I had not seen this before, I was not aware. Thanks for letting me know about it.

In terms of other options, There isn't really many alternatives that are chromium based, are there? Brave has crypto garbage in it you have to manually disable, vanadium doesn't do ad-blocking. Most other popular mobile browsers are all Gecko-based.

**Comment by severtheskyline** [comment]
Damn, you don't know what you have until you loose it. Probably the first app I installed in any device, gonna miss you bromite, big sad

**Comment by Spark4000** [comment]
Looks like Carl commented yesterday. https://github.com/bromite/bromite/pull/2561

**Comment by sankhababu** [comment]
> Looks like Carl commented yesterday. #2561

Great information. Thanks for this update. 👏

**Comment by BabyOilJohnson** [comment]
Somethings [cooking up](https://github.com/uazo/cromite).

**Comment by BabyOilJohnson** [comment]
> Somethings [cooking up](https://github.com/uazo/cromite).

It's on.

**Comment by d1stru3t0r** [comment]
> > Somethings [cooking up](https://github.com/uazo/cromite).
> 
> It's on.

But still no SystemWebView which was mostly used :(

**Comment by SlimTux** [comment]
[EDIT]
---- 
# INVALID MESSAGE(See my message bellow @HarriBuh ), cromite is a **FORK** not made by the official (bromite)creator/dev.
# The project did not migrate to Cromite
----
[Old Message bellow]
### YES!/no
#### The project has migrated to ["Cromite"](https://github.com/uazo/cromite)      (The same developer's project)  ![bromite-dead](https://github.com/bromite/bromite/assets/117015097/28b2a787-b491-4fc1-9f98-5fa99bfd3e48)

**Comment by HarriBuh** [comment]
@SlimTux It's definitely not the same developer, Uazo was a contributor here. Cromite is his own project now.

**Comment by SlimTux** [comment]
> @SlimTux It's definitely not the same developer, Uazo was a contributor here. Cromite is his own project now.

Sorry for that. I did not realize they were different people. Please disconsider my previous message. 👍
